src/Wallabag/ImportBundle/Import/AbstractImport.php

```diff
@@ -89,11 +91,15 @@ public function getMarkAsRead()
      */
     protected function fetchContent(Entry $entry, $url, array $content = [])
     {
-        try {
-            return $this->contentProxy->updateEntry($entry, $url, $content);
-        } catch (\Exception $e) {
+        $updatedEntry = clone $entry;
+        $this->contentProxy->updateEntry($updatedEntry, $url, $content);
+        if ($updatedEntry->getContent() === $this->fetchingErrorMessage) {
+            $this->logger->warn('Unable to update imported entry ('.$entry->getUrl().')');
+
             return $entry;
         }
+
+        return $updatedEntry;
```

Comment:
I think this whole stuff shouldn't be in AbstractImport because when it fails, you are re-doing thing that are already perform in updateEntry (like setting the domain name).
Maybe it should be better to move this logic in ContentProxy